<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherWidget</class>
 <widget class="QWidget" name="WeatherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="degreeValue">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>109</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>150</pointsize>
    </font>
   </property>
   <property name="text">
    <string>55</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="precipitationLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>110</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Pericipitation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="humidityLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>143</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Humidity:</string>
   </property>
  </widget>
  <widget class="QLabel" name="windLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>176</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Wind:</string>
   </property>
  </widget>
  <widget class="QLabel" name="humidityValue">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>143</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="windValue">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>176</y>
     <width>170</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 mph dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="locationValue">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>20</y>
     <width>505</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>38</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>LOCATION</string>
   </property>
  </widget>
  <widget class="QLabel" name="precipitationValue">
   <property name="geometry">
    <rect>
     <x>689</x>
     <y>110</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 in</string>
   </property>
  </widget>
  <widget class="QSvgWidget" name="weatherIconContainer" native="true">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>-30</y>
     <width>375</width>
     <height>375</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="cloudCoverLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>209</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Cloud Cover:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cloudCoverValue">
   <property name="geometry">
    <rect>
     <x>669</x>
     <y>209</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0%</string>
   </property>
  </widget>
  <widget class="QLabel" name="visibilityLabel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>242</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Visibility:</string>
   </property>
  </widget>
  <widget class="QLabel" name="visibilityValue">
   <property name="geometry">
    <rect>
     <x>629</x>
     <y>242</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0 mi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>90</y>
     <width>505</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #fff;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSvgWidget</class>
   <extends>QWidget</extends>
   <header>qsvgwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
